dropshot/src/config.rs

```diff
@@ -113,9 +116,108 @@ impl Default for ConfigDropshot {
     fn default() -> Self {
         ConfigDropshot {
             bind_address: "127.0.0.1:0".parse().unwrap(),
-            request_body_max_bytes: 1024,
+            default_request_body_max_bytes: 1024,
             default_handler_task_mode: HandlerTaskMode::Detached,
             log_headers: Default::default(),
         }
     }
 }
+
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
```

Comment:
does this need to be Serialize?

Reply:
This is coupled with the `into` comment.